aspnetcore/security/authentication/index.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Overview of ASP.NET Core Authentication
+author: mjrousos
+description: Learn about authentication in ASP.NET Core.
+ms.author: riande
+ms.custom: mvc
+ms.date: 11/22/2019
+uid: security/authentication/index
+---
+# Overview of ASP.NET Core authentication
+
+By [Mike Rousos](https://github.com/mjrousos)
+
+Authentication is the process of determining a user's identity. [Authorization](xref:security/authorization/introduction) is the process of determining whether a user has access to a resource. In ASP.NET Core, authentication is handled by authentication [middleware](xref:fundamentals/middleware/index). The authentication middleware uses registered authentication handlers to complete authentication-related actions. Examples of authentication-related actions include:
+
+* Authenticating a user.
+* Responding when an unauthenticated user tries to access a restricted resource.
+
+The registered authentication handlers and their configuration options, are called "schemes".
+
+Authentication schemes are specified by registering authentication services in `Startup.ConfigureServices`. This is done by:
+
+* Calling [AuthenticationBuilder.AddScheme](xref:Microsoft.AspNetCore.Authentication.AuthenticationBuilder.AddScheme*)
```

Comment:
Should we put the more common way first?

Reply:
My thought was to put the lower level way first (so there's less behind the scenes magic) and then call out the more common way.

Reply:
I've modified this to put the more common way first.